src/components/UserFeedbackComponent.jsx

```diff
@@ -109,7 +91,7 @@ export class UserFeedbackComponent extends React.Component {
             <Table.Row>
               <Table.HeaderCell colSpan="5">
                 {
-                  this.props.feedback.length === 0 ? '' :
+                  !(this.props.feedbackCount === 0) &&
```

Comment:
This can be done in `mapStateToProps` and can be evaluated with `!!` as 0 is *falsy* and any number would be coerced to a boolean.  E.g.
```
const mapStateToProps = ({ feedbackReducer }) => {
  ...
  ...
  return {
    ...
    ...
    hasFeedback: !!feedbackCount
  }